Zend/zend_vm_def.h

```diff
@@ -3207,7 +3207,7 @@ ZEND_VM_HANDLER(70, ZEND_FREE, TMPVAR, ANY)
 	USE_OPLINE
 
 	SAVE_OPLINE();
-	zval_ptr_dtor_nogc(EX_VAR(opline->op1.var));
+	FREE_OP1();
```

Comment:
You don't really need this change, the handler will use `zval_ptr_dtor_nogc(EX_VAR(opline->op1.var));` already (because it's TMPVAR).
OTOH it doesn't hurt.

Reply:
Yes, I left that in because it is more generically correct (without resulting in a change to the generated parts). But I'm happy to revert if an unrelated change is undesired.

Reply:
I think it's cleaner if the commit history only contains the changes related to this PR. I don't care too much but I would have a slight preference if you committed this separately to master. If you were touching this function anyway and you did this as a drive-by change I'd be fine with it.
My main motivation for doing it separately is that now when doing a git blame this will pop up as part of this PR.

Reply:
I generally agree with not bundling unrelated changes in a PR for the same reasons. I've spun out the change into #18131.